streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBTimeOrderedKeyValueBufferTest.java

```diff
@@ -62,18 +62,16 @@ public class RocksDBTimeOrderedKeyValueBufferTest {
     @SuppressWarnings({"rawtypes", "unchecked"})
```

Comment:
Sorry for the delayed feedback. To address the following warnings, could you please move these suppression annotations to the class-level?
```
> Task :streams:compileTestJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /home/chia7712/project/kafka/streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBTimeOrderedKeyValueBufferTest.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
```

Reply:
Nice catch! -- I think we should _never_ use class level suppression though. Opened a PR to suppress for the individual methods: https://github.com/apache/kafka/pull/18580